src/schemas/dataset.schema.json

```
@@ -75,6 +75,7 @@
         }
       }
     },
-    "staticLayout": { "$ref": "#/definitions/components" }
+    "staticLayout": { "$ref": "#/definitions/components" },
+    "theme": { "type": "string" }
```

```suggestion
    "theme": {
      "type": "string",
      "enum": ["dark", "light"]
     }
```
or something like that.